experimentation/src/main/java/com/automattic/android/experimentation/ExPlat.kt

```diff
@@ -65,32 +67,29 @@ public class ExPlat internal constructor(
         coroutineScope.launch { repository.clearCache() }
     }
 
-    private fun refresh(refreshStrategy: RefreshStrategy) {
-        if (experimentIdentifiers.isNotEmpty()) {
-            getAssignments(refreshStrategy)
-        }
-    }
+    private fun getAssignments(): Assignments? = repository.getCached()
 
-    private fun getAssignments(refreshStrategy: RefreshStrategy): Assignments? {
+    private fun invalidateCache(): Assignments? {
```

Comment:
Not really clear to me why `invalidateCache()` may return cached assignments?
I mean, even if they're not stale, a function to invalidate them should force (now or on the next fetch of the cache) the cache creation, no? 🤔

Reply:
Of course you're right, thank you for spotting this 🙌 Addressed in 3dd77fa0b568d43110c3d5ced8af22050659a02b